experiment/project.clj

```diff
@@ -1,7 +1,46 @@
 (defproject thinktopic/experiment "0.9.4-SNAPSHOT"
-  :description "FIXME: write description"
-  :url "http://example.com/FIXME"
-  :license {:name "Eclipse Public License"
-            :url "http://www.eclipse.org/legal/epl-v10.html"}
+  :description "A higher-level library for performing experiments with cortex."
```

Comment:
experiment seems a little generic when this is so tied to (and a part of) cortex. `cortex-experiment` seems a bit more apt, imho.